ds_judgements_public_ui/sass/includes/_cards_homepage.scss

```diff
@@ -1,89 +1,208 @@
 .cards-homepage {
   display: flex;
-  flex-direction: column;
-  gap: 0;
+  flex-direction: row;
+  gap: $space-4 !important;
 
-  margin: $space-4 auto;
+  margin: 0;
+  margin-top: $space-10;
   padding: 0;
 
   list-style-type: none;
 
+  &:hover {
+    text-decoration: none;
+  }
+
   li {
-    @media (max-width: $grid-breakpoint-medium) {
-      height: 115px;
-      margin: 0 auto;
+    @media (width <= 1199px) {
+      height: $space-8;
+      margin: 0;
+      margin-bottom: $space-5;
+    }
+  }
+
+  &__container {
+    @include container;
+
+    display: flex;
+    flex-direction: column;
+    gap: $space-4;
+    padding-top: $space-8;
+
+    h2 {
+      display: none;
+
+      @media (width <= 1199px) {
+        @include heading_text;
+        @include header($typography-header-h2-size);
+
+        display: contents;
+      }
+
+      @media (max-width: $grid-breakpoint-small) {
+        display: contents;
+        margin-top: 0;
+      }
+    }
+
+    @media (width <= 1199px) {
+      gap: 0;
+      align-items: flex-start;
+
+      margin-top: $space-5;
+      margin-right: $space-5;
+      margin-left: $space-5;
+      padding: $space-2 $space-4 $space-8 !important;
+
+      border: 3px solid $color-yellow;
+    }
+
+    @media (max-width: $grid-breakpoint-large) {
+      padding: $space-3;
+    }
+
+    @media (max-width: $grid-breakpoint-extra-large) {
+      padding: $space-7;
     }
   }
 
   @media (min-width: $grid-breakpoint-small) {
     gap: 0;
   }
 
-  @include container;
+  @media (width <= 1199px) {
+    float: left;
+    flex-direction: column;
+    gap: 0 !important;
+  }
 }
 
 .card-homepage {
   display: flex;
-  gap: $space-4;
+  gap: $space-2;
+
   margin: $space-4 $space-8;
+  padding: 0;
+
+  text-decoration: underline;
+  text-decoration-color: colour-var("accent-background-light");
+
+  background-color: colour-var("accent-background-light");
+  border-top: 4px solid $color-yellow;
+
+  &:hover {
+    text-decoration: underline;
+    text-decoration-color: colour-var("accent-background");
+    background-color: colour-var("accent-background");
+  }
 
   h3 {
     margin-top: auto;
     margin-bottom: 0;
-    font-weight: $typography-bold-font-weight;
+
+    color: colour-var("link");
+    text-align: left;
+    text-decoration: underline !important;
+    text-decoration-color: colour-var("accent-background-light");
+    text-wrap: wrap;
 
     a {
       color: colour-var("link");
+      text-decoration: none;
     }
 
-    @media only screen and (max-width: $grid-breakpoint-medium) {
+    @media only screen and (width <= 1199px) {
       margin-top: $space-2;
 
-      font-size: $typography-md-text-size;
+      font-size: $typography-lg-text-size;
       line-height: $typography-sm-line-height;
       text-align: left;
+      text-decoration: underline;
+      text-decoration-color: colour-var("link");
       text-wrap: balance;
```

Comment:
There are some of these `text-wrap: balance` statements which actually belong outside of the media query, eg:

<img width="389" alt="Screenshot 2024-12-12 at 14 00 27" src="https://github.com/user-attachments/assets/c5499019-9c16-4f0c-acec-5ef48d800ab8" />

should break before the "or".